haret/src/vr/states/common/normal.rs

```diff
@@ -67,12 +67,11 @@ pub fn handle_start_view<T: State>(state: T,
 }
 
 pub fn handle_get_state<T: State>(state: T,
-                                  msg: GetState,
+                                  GetState { epoch, view, op }: GetState,
```

Comment:
Wow, I swear I tried this a couple years ago and failed. I must have "Done it Wrong"™